Sources/SwiftJavaRuntimeSupport/DefaultCaches.swift

```diff
@@ -47,6 +47,36 @@ extension _JNIMethodIDCache {
     }
   }
 
```

Comment:
I just noticed the filename here, let's please call this `JNIMethodIDCaches` "default caches" is too ambiguous